wandb/sdk/service/streams.py

```diff
@@ -41,6 +44,7 @@ class StreamRecord:
     _relay_q: "multiprocessing.Queue[pb.Result]"
     _iface: InterfaceRelay
     _thread: StreamThread
+    _settings: Settings
```

Comment:
as discussed, this can be a SettingsStatic object for now i think... this way we can defer some of this Settings refactor stuff until a later PR.

Reply:
will do and I think I'll leave a setting todo here